src/mcp_server/tools/observability_vllm_tools.py

```diff
@@ -223,6 +232,8 @@ def analyze_vllm(
     time_range: Optional[str] = None,
     start_datetime: Optional[str] = None,
     end_datetime: Optional[str] = None,
+    start_ts: Optional[int] = None,
```

Comment:
Why do we need both start_datetime and start_ts, and end_datetime & end_ys in the tools?

Reply:
@redhatHameed as we discussed, have both start_datetime and start_ts can be confusing for the MCP server users later on and we have to define which one takes priority if both are supplied. So for simplicity we can just use start_datetime, and change UI code to convert the timestamp to the string and then call the MCP tools.

Reply:
@jianrongzhang89 thanks done the change.